README.md

```diff
@@ -1,5 +1,3 @@
-[![Build Status](https://travis-ci.org/google/google-api-dotnet-client.svg?branch=master)](https://travis-ci.org/google/google-api-dotnet-client)
```

Comment:
Well crap. I've been having a lot of issues since I tried doing the rebasing, my self-taught is showing. Is there a way I can undo/ignore this? Alternately I've been considering just making a new fork and manually adding the changes below to it, there aren't that many and it could be done in decent time. I'm not sure if that's the 'proper' way to fix it, but... horseshoes and hand grenades this is not.